website/static/js/profile.js

```diff
@@ -468,19 +493,35 @@ var SocialViewModel = function(urls, modes) {
     var self = this;
     BaseViewModel.call(self, urls, modes);
     TrackedMixin.call(self);
+    var i;
 
     self.addons = ko.observableArray();
+  
+    self.profileWebsites = ko.observableArray(); // Initially a single item with a blank first entry  
```

Comment:
Why populate this array with a blank item? I think the array can just contain the added entries.